main.go

```diff
@@ -85,6 +86,27 @@ func checkUnused(deps []depEntry, vd string) {
 	}
 }
 
+func checkLicense(deps []depEntry, vd string) {
+	for _, d := range deps {
+		dir := filepath.Join(vd, d.importPath)
+		files, err := ioutil.ReadDir(dir)
+		if err != nil {
+			// err can be nil for unused package
+			continue
+		}
+		licenseFiles := 0
+		for _, file := range files {
+			p := filepath.Join(dir, file.Name())
+			if isLicenseFile(p) {
```

Comment:
Detecting README is intentional (https://github.com/LK4D4/vndr/pull/21), but maybe it is good to `grep -i license README*` in another PR